### ISSUE TITLE
Delete Job Reversal

### ISSUE THREAD

**Issue by itdependsnetworks** [nautobot/nautobot#2325]
### As ...

Patti - Platform Admin

### I want ...

to be able to recover a job that I have deleted

### So that ...

I can get jobs back that were once deleted

### I know this is done when...

I have a way or documentation the "reinstalled or reintroduced" process. 

### Optional - Feature groups this request pertains to.

- [ ] Automation
- [ ] Circuits
- [ ] DCIM
- [ ] IPAM
- [ ] Misc (including Data Sources)
- [ ] Organization
- [ ] Plugins (and other Extensibility)
- [ ] Security (Secrets, etc)
- [ ] Image Management
- [ ] UI/UX
- [ ] Documentation
- [ ] Other (not directly a platform feature)

### Database Changes

No

### External Dependencies

N/A

**Comment by itdependsnetworks** [comment]
The documentation refers to: 
```
An administrator or user with extras.delete_job permissions may delete such a Job database record if desired, but be aware that doing so will result in any existing JobResult or ScheduledJob records that originated from this Job losing their association to the Job; this association will not be automatically restored even if the Job is later reinstalled or reintroduced.
```

But not sure how to `reinstalled or reintroduced` it?

**Comment by glennmatthews** [comment]
Agreed that we should document this. In general it's going to be the same process as if the job were a new one, i.e. re-sync the Git repository or re-run `nautobot-server post_upgrade`.

**Comment by itdependsnetworks** [comment]
I think clarification should be added to the delete that it "won't stick" if the job still exists.
